BrigetteDoelp/fitlit-starter-kit
# Create & Test User and User Repo

### What’s this PR do? 
- Creates the UserRepo class
- Creates User class
- Implements testing suites for both classes and methods on both classes
### Where should the reviewer start?
Start on `userrepo-test.js`, then move into `UserRepo.js`, making sure the tests are written well and are actually testing what needs to be tested. 
Then, move into `user-test.js`, once again checking for accuracy of tests. Move into `User.js`, repeating the same process.
Along the way. please check for typos, and any inconsistencies in code punctuation. 
### How should this be manually tested?
Simply run npm test in the base of the repository, or if you choose not to clone it down, simply make sure that all of the logic in the methods tracks.​
### Any background context you want to provide? 
- ​ Files got tests and test data were all created in this pull request. ​​
### Relevant tickets?
resolves #2 
resolves #3 
resolves #4 
revolves #5 
resolves #6 

​
